Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.10.8
+
+RUN useradd --create-home --shell /bin/bash bananabot
+
+USER bananabot
+RUN pip install Flask Flask-HTTPAuth requests
```

Comment:
List dependencies within a `requirements.txt` file instead saving the hassle for anyone trying to configure their own environment.